src/linker/pipeline.py

```diff
@@ -91,13 +92,20 @@ def build_snakefile(self) -> Path:
             self.snakefile_path.unlink()
         self.write_imports()
         self.write_target_rules()
+        if self.config.spark:
+            self.write_spark_module()
```

Comment:
We talked about this already but adding here for posterity: this will set up a spark cluster that lasts for the entire pipeline as opposed to the previous behavior of spinning up a cluster per step.